iaso/api/mobile/entity.py

```diff
@@ -59,7 +99,9 @@ class Meta:
 
     @staticmethod
     def get_instances(entity: Entity):
-        return list(map(lambda instance: MobileEntityAttributesSerializer(instance).data, entity.non_deleted_instances))  # type: ignore
+        return list(
+            map(lambda instance: MobileEntityAttributesSerializer(instance).data, entity.non_deleted_instances)  # type: ignore
+        )
```

Comment:
@olethanh told me this should be written like this:

```suggestion
        return [ MobileEntityAttributesSerializer(instance).data for instance in entity.non_deleted_instances ]
```

Reply:
actually you would write `return MobileEntityAttributesSerializer(entity.non_deleted_instances, many=True).data`

Reply:
actually rreading the whole code you don't even need a function

`instances = MobileEntityAttributesSerializer(many=True, source="non_deleted_instances")` should suffice I think